decidim-conferences/app/views/decidim/conferences/media/index.html.erb

```diff
@@ -9,23 +9,34 @@ edit_link(
 )
 %>
 
-<div class="row">
-  <div class="columns mediumlarge-12 conference-media">
-    <% if @media_links.any? %>
-      <section>
-        <h3 class="section-heading"><%= t("media.index.title", scope: "decidim") %></h3>
-        <p><%= t("media.index.description", scope: "decidim") %></p>
-      </section>
-      <hr class="reset mt-none mb-s">
-      <div class="media-links">
-        <ul class="list-reset">
-          <% @media_links.each do |media_link| %>
-            <%= cell "decidim/conferences/media_link", media_link %>
-          <% end %>
-        </ul>
+<%# NOTE: this page does not use a regular layout %>
+<div class="container">
```

Comment:
You have an accessibility violation currently on this page:
![Accessibility violation](https://user-images.githubusercontent.com/864340/200785141-952c088f-7f10-4241-a90b-d0e20409ff6b.png)